news/2021-08-23-summer-tf-fanart-contest-results.md

```diff
@@ -0,0 +1,109 @@
+
+[![](https://assets.ppy.sh/contests/133/banners/kabandayo.jpg)](https://assets.ppy.sh/contests/133/winners/kabandayo.png)
+
+[**kabandayo**](https://osu.ppy.sh/users/21567829) wowed over **1472** voters with their take on a baton pass moment between pippi and Mocha mid way through a race. Just look at how excited pippi is!
+
+[![](https://assets.ppy.sh/contests/133/banners/Tin%20Can.jpg)](https://assets.ppy.sh/contests/133/winners/Tin%20Can.png)
+
+**1222** voters joined Chirou, a don and the Taikonator in cheering on [**Tin Can**](https://osu.ppy.sh/users/10682807)'s pippi and Mocha through a baton marathon. Unfortunately for Mocha, it seems like she's about to be introduced to the ground or drop her baton — possibly both at the same time. Such is the struggle of an athlete.
+
+[![](https://assets.ppy.sh/contests/133/banners/Sutsuka.jpg)](https://assets.ppy.sh/contests/133/winners/Sutsuka.png)
+
+In an EXTREMELY unexpected twist, [**Sutsuka**](https://osu.ppy.sh/users/29089) left **1200** voters awestruck with their incredible take on osu! themed ancient Greek terracotta art, including a blazing prize amphora sporting all kinds of osu! game mode inspired motifs.
+
+If you're at all familiar with this kind of historical art, you'll appreciate the incredible attention to detail that went into this piece, all while embracing the mascots you know and love.
+
+A truly spectacular piece and easily one of the most memorable fanart entries I think I have ever seen thus far. Wow.
+
+[![](https://assets.ppy.sh/contests/133/banners/Loichuu.jpg)](https://assets.ppy.sh/contests/133/winners/Loichuu.png)
+
+**1159** voters embraced the chaos (and cute) of [**Loichuu**](https://osu.ppy.sh/users/460210)'s most athletic scene depicting pippi undertaking a victory lap while Chirou drops her ice water bowl in an ill-fated pursuit. And of course, where there is Yuzu, there is *shenanigans*, as that rather ominously aimed disc might suggest...
+
+---
+
+[Download a zip file containing all the winning entries in high definition here!](https://assets.ppy.sh/contests/133/winners/Summer2021TFWinners.zip)
+
+The [Summer 2021 Track and Field Fanart Contest listing](https://osu.ppy.sh/community/contests/133) awaits with the full rundown for everyone who entered, so make sure to check that all out and see where your favourite entries ended up in the tally if they didn't make it here.
+
+You'll find the finalists listed above are available in game now as the Summer 2021 seasonal backgrounds (and in osu!lazer too!). The top three artists have also earned themselves a special profile badge featuring their winning work, and everybody here also goes home with 3 months of osu!supporter to celebrate their victory.
+
+Thank you to everybody who entered and voted, and we'll see you in only a couple of weeks for another Fan Art contest, because as you all know by now, the end of the year is just chock full of them.
```

Comment:
```suggestion
Thank you to everybody who entered and voted, and we'll see you in only a couple of weeks for another Fan Art contest, because as you all know by now, the end of the year is just chock-full of them.
```